src/stats.rs

```diff
@@ -68,6 +70,8 @@ pub struct PeerStats {
     pub timestamp: Instant,
     /// The last egress bandwidth estimate from the BWE subsystem, if enabled.
     pub bwe_tx: Option<Bitrate>,
+    /// The egress loss over the last second expressed as a percentage value between 0.0 and 1.0.
```

Comment:
Fraction or ratio?

Reply:
or just multiply by 100 and make it an u8?

Reply:
Hmm, I think keeping it as f64 would be nice but I can reword it

Reply:
f32?

Reply:
No preference, I just defaulted to f64. Do we have a stated preference for the project?

Reply:
i think it depends on what we're talking about. a percentage number between 0.0-1.0 might not need the precision of f64.

Reply:
Fair, f32 should be sufficient. Will change

Reply:
also if you say egress_loss_ratio it is clear that you return a ratio and I will expect numbers between 0 and 1

Reply:
> also if you say egress_loss_ratio it is clear that you return a ratio and I will expect numbers between 0 and 1

Good point, although I know @algesten isn't a fan of long names :D

Reply:
comment still percentage